cmd/down.go

```diff
@@ -9,7 +9,7 @@ import (
 
 var downCmd = &cobra.Command{
 	Use:   "down",
-	Short: "A brief description of your command",
+	Short: "Stop and remove containers, networks, images, and volumes running on K8s",
```

Comment:
`Deletes the pods of the specified docker compose services. If all docker compose services would be deleted then the Kubernetes services are also deleted.`